regions_www/m/pct_shiny_funs.R

```diff
@@ -504,6 +524,10 @@ popup_zones <- function(data, scenario, purpose){
   }
 
   } else if (purpose == "school") {
+
+    # Create a new variable called font_colour which changes into red colour when change in mMET is negative
+    data@data$font_colour <- ifelse(round(data[[data_filter(scenario, "simmet")]]) < 0, "red", "black")
```

Comment:
That is a nice addition.  Is there a reason you are adding it to `data` (and `round`ing) instead of

```r
font_colour <- ifelse(data[[data_filter(scenario, "simmet")]] < 0, "red", "black")
```

Reply:
this wasn't my addition, it has been there for at least a year in the commuting layer (e.g. line 268 here https://github.com/npct/pct-shiny/blob/3a06d3c6ffb42fd0ac23dfe71a50f0ce6b656b78/pct-shiny-funs.R), I was just copying...

I guess the reason to use rounding is that we also round the variable it applies to, and we didn't want have some zeros be black and some red but it not be clear why they were different (since one could not know behind-the-scenes that one was +0.001 and the other -0.001).  I have made a tweak, since I realise that the commuting layer it was correct to use 'round' ( since we were dealing in variables with large units) but for the school layer it is better to use round' and specify 3dp, as that is what is being used for the school variable 

As to why it is being added to data as a variable... I don't know why that was done originally. I would be tempted to leave it, just on the basis that it seems to be working, but equally would be happy for you to have a go at improving